plugins/modules/proxmox.py

```diff
@@ -790,6 +814,15 @@ def main():
                 module.exit_json(changed=True, msg="VM %s is shutting down" % vmid)
         except Exception as e:
             module.fail_json(msg="stopping of VM %s failed with exception: %s" % (vmid, e))
+    
+    elif state == 'template':
+        try:
+            vm = proxmox.get_vm(vmid)
+
+            if proxmox.convert_to_template(vm, vmid, timeout, force=module.params['force']):
+                module.exit_json(changed=True, msg="VM %s is converted to template" % vmid)
```

Comment:
Since the function can only return `True` (if it doesn't raise an exception), I would get rid of the `if` since it implies to the reader that it is also possible that it returns `False`:
```suggestion
            proxmox.convert_to_template(vm, vmid, timeout, force=module.params['force'])
            module.exit_json(changed=True, msg="VM %s is converted to template" % vmid)
```